content/design-guidelines/icons-and-illustrations.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Icons and illustration usage
+---
+
+import {Text} from '@primer/components'
+
+## Guidelines
+
+Below are guidelines on how to best use GitHub icons and illustrations within a presentation. Approved visual assets are provided within the [asset library](https://primer.style/presentations/assets).
+
+<DoDontContainer>
+  <Dont>
+    <Text>Alter the illustrations</Text>
+  </Dont>
+  <Dont>
+    <Text>Combine images that use different stylings</Text>
+  </Dont>
+</DoDontContainer>
+
+## Icons
+
+Icons are used to show a visual association with different objects (e.g., The core four include: Developers, Teams, Enterprise, and Maintainers). There are two sets of icons specific to GitHub: the [GitHub marketing icons](https://ghicons.github.com/) and the [Octicons](https://octicons.github.com/).
```

Comment:
I don't understand what the content inside the brackets means, but the GH styleguide is to use "example" and not "e.g."
```suggestion
Icons are used to show a visual association with different objects (e.g., The core four include: Developers, Teams, Enterprise, and Maintainers). 

There are two sets of icons specific to GitHub: 
- [GitHub marketing icons](https://ghicons.github.com/)
- [Octicons](https://octicons.github.com/)
```